x/ibc-transfer/client/cli/tx.go

```diff
@@ -10,28 +11,19 @@ import (
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
 	authclient "github.com/cosmos/cosmos-sdk/x/auth/client"
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
 	"github.com/cosmos/cosmos-sdk/x/ibc-transfer/types"
 )
 
-// IBC transfer flags
-var (
-	FlagNode1    = "node1"
-	FlagNode2    = "node2"
-	FlagFrom1    = "from1"
-	FlagFrom2    = "from2"
-	FlagChainID2 = "chain-id2"
-	FlagSequence = "packet-sequence"
-	FlagTimeout  = "timeout"
-)
-
 // GetTransferTxCmd returns the command to create a NewMsgTransfer transaction
 func GetTransferTxCmd(cdc *codec.Codec) *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "transfer [src-port] [src-channel] [dest-height] [receiver] [amount]",
-		Short: "Transfer fungible token through IBC",
-		Args:  cobra.ExactArgs(5),
+	return &cobra.Command{
+		Use:     "transfer [src-port] [src-channel] [receiver] [amount] [timeout-height] [timeout-timestamp]",
```

Comment:
I think it would be nice to make the timeouts optional flags, which can lookup a default value